install/resources/upgrade44.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
We already have an upgrade script for 1.8.16: https://github.com/mybb/mybb/blob/feature/install/resources/upgrade43.php - the naming is super confusing, I know 😞

Reply:
Was not familiar with upgrade script system. Sorry. Will merge to that.

Reply:
@euantorano 44 unlinked. code transferred to 43